composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "signalwire/signalwire",
+  "name": "aloware/signalwire",
```

Comment:
Do we need to change this? Considering we might want to push back the codes into the original repo.

Reply:
oh i see.. okay ill revert it..